include/fastdds/rtps/participant/RTPSParticipant.h

```diff
@@ -299,6 +299,11 @@ class RTPS_DllAPI RTPSParticipant
     bool ignore_reader(
             const GUID_t& reader_guid);
 
+    /**
+     * Returns registered transports' netmask filter information (transport's netmask filter kind and allowlist).
+     */
+    std::vector<fastdds::rtps::TransportNetmaskFilterInfo> netmask_filter_info() const;
```

Comment:
NIT, I may go with a `get`
```suggestion
    std::vector<fastdds::rtps::TransportNetmaskFilterInfo> get_netmask_filter_info() const;
```